### ISSUE TITLE
Server returns 500 when trying to validate my add-on

### ISSUE THREAD

**Issue by csergiu** [mozilla/addons-server#10180]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)
When I go to [https://addons.mozilla.org/en-US/developers/addon/burner-emails/versions/submit/](https://addons.mozilla.org/en-US/developers/addon/burner-emails/versions/submit/) and I try to upload my .zip file with my add-on, the server returns 500:

> Your add-on failed validation with 1 error.
> 
> • Received an empty response from the server; status: 500

### What happened?
Validation server returned 500.

### What did you expect to happen?
Validation server returned 2 warnings about my code like last 30 times and I could upload my add-on...

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)
Nothing comes to mind right now.

**Comment by wagnerand** [comment]
Can you please attach or link to the file you are trying to submit?

**Comment by csergiu** [comment]
@wagnerand Yup, here it is.
[firefox.zip](https://github.com/mozilla/addons-server/files/2654908/firefox.zip)

**Comment by wagnerand** [comment]
Thank you, that allowed us to identify the problem: https://sentry.prod.mozaws.net/operations/olympia-prod/issues/4998011/

tl;dr: The emoji in the description of your manifest ("Detects email fields as you browse and fills them with a burner email address that forwards to your personal email address 😎") is the culprit.

@eviljeff that description is inserted into `akismet_reports.comment`, which is a "LONGTEXT". Is this issue part of/caused by #8923? Anything we can do to prevent the 500 or at least give a better error message?

**Comment by wagnerand** [comment]
@csergiu In the meantime, you could work around this issue by removing the emojis in your locale file and submitting a new version.

**Comment by csergiu** [comment]
@wagnerand Thanks! I'll do that.

**Comment by diox** [comment]
Duplicate of #8923
